### PR TITLE
Update JTBD survey

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Running for a second time, I removed the dates stuff because it is already handled by the targeting tool.

### CODE REVIEW

File: static/src/javascripts/projects/journalism/jtbd-survey.js [hunk 1]
```diff
@@ -139,9 +134,7 @@ const init = (): void => {
 
     mediator.emit('journalism:modules:jtbd', true);
 
-    localStorage.setIfNotExists('gu.jtbd.questions', qs, {
-        expires: endOfSurvey,
-    });
+    localStorage.setIfNotExists('gu.jtbd-updated.questions', qs);
```
Comment (annebyrne) [inline static/src/javascripts/projects/journalism/jtbd-survey.js]: why doesn't it expire at end of survey anymore?

**Comment by annebyrne** [comment]
Just one question about expiry date for my own curiosity, but other than that LGTM!

File: static/src/javascripts/projects/journalism/jtbd-survey.js [hunk 2]
```diff
@@ -76,14 +70,14 @@ const save = (
     as: Answer[],
     q: Question,
     answer: Answer,
-    why: ?string
```
Comment (tomrf1) [inline static/src/javascripts/projects/journalism/jtbd-survey.js]: `why?: string` is an optional param. What was the effect of `why: ?string`?

**Comment by tomrf1** [comment]
👍

File: static/src/javascripts/projects/journalism/jtbd-survey.js (hunk 1 above)
Comment (regiskuckaertz) [reply static/src/javascripts/projects/journalism/jtbd-survey.js]: the expiry option in the storage module is only a working if there is someone requesting a value _after_ it has expired. in other words, it does not prevent a value to hang in there forever. afaik, the only place where you can specifically address this issue is here:

https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/common.js#L126-L135

File: static/src/javascripts/projects/journalism/jtbd-survey.js (hunk 2 above)
Comment (regiskuckaertz) [reply static/src/javascripts/projects/journalism/jtbd-survey.js]: the latter accepts `null` as a value. but in that case, it doesn't really matter. I don't know why I did this, will remove it 😋

**Pull request merged**